libs/sdk-utils/multicall/src/lib/multicall-logging.integration.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { logDirectory, loggerFactory, LOG_FILE_NAMES } from './multicall-logging';
+import fs from 'fs';
+import { exec } from 'child_process';
+import { promisify } from 'node:util'
```

Comment:
; in the end of line missing, also around this file sometimes you use it sometimes not